resources/lang/fr/buttons.php

```diff
@@ -18,14 +18,14 @@
             'users' => [
                 'activate'           => 'Activer',
                 'change_password'    => 'Changer de mot de passe',
-                'clear_session'         => 'Clear Session',
-                'confirm'             => 'Confirm',
+                'clear_session'       => 'Effacer la session',
```

Comment:
Hi,
I do not feel still comfortable in use of git for the moment.
Did i need to do some changes?

Regards.